digdag-ui/model.js

```diff
@@ -297,6 +297,15 @@ export class Model {
     })
   }
 
+  retrySession (session: Session) {
+    const { lastAttempt } = session
+    return this.put('attempts', {
```

Comment:
I believe `retryAttemptName` must also be set to a unique name. There is a unique constrain on `(session_id, attempt_name)` in the `session_attempts` table.

https://github.com/treasure-data/digdag/blob/master/digdag-core/src/main/java/io/digdag/core/database/migrate/Migration_20151204221156_CreateTables.java#L120

The digdag cli generates a unique retry attempt name for the user if one has not been specified.

https://github.com/treasure-data/digdag/blob/master/digdag-cli/src/main/java/io/digdag/cli/client/Retry.java#L123